test/source/tests/setup.ts

```diff
@@ -41,6 +40,27 @@ export const defineSetupTests = (testVariant: TestVariant, testWithBrowser: Test
       await settingsPage.notPresent('.settings-banner');
     }));
 
+    ava.default('setup - imported key with multiple alias should show checkbox per alias', testWithBrowser(undefined, async (t, browser) => {
```

Comment:
@martgil So we just have to move this test out of `if (testVariant !== 'CONSUMER-LIVE-GMAIL') {...}` clause and put it at the bottom of this file enclosed in `if (testVariant === 'CONSUMER-MOCK') {...}` so it won't be executed on the enterprise mock.